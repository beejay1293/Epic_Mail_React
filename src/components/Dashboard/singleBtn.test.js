import React from 'react';
import { mount, shallow, render } from 'enzyme';
import SingleBtn from './SingleBtn';
describe('Single button', () => {
  it('should render correctly in single button with props', () => {
    const SBtn = shallow(<SingleBtn styles= 'inbox__btn active' name='Inbox' font='fas fa-inbox'/>);
  
    expect(SBtn).toMatchSnapshot();
  });
});
