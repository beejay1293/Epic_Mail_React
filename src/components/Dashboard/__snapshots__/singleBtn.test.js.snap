// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single button should render correctly in single button with props 1`] = `
<li
  className="inbox__btn active"
>
  <i
    className="fas fa-inbox"
  />
  Inbox
</li>
`;
